table/block_based/block_based_table_builder.cc

```diff
@@ -369,6 +367,35 @@ struct BlockBasedTableBuilder::Rep {
   uint64_t get_offset() { return offset.load(std::memory_order_relaxed); }
   void set_offset(uint64_t o) { offset.store(o, std::memory_order_relaxed); }
 
+  bool ok() const {
+    assert(io_status.ok() || !status.ok());
+    return status.ok();
+  }
+
+  const IOStatus& get_io_status() const { return io_status; }
```

Comment:
Naming style

Reply:
Google C++ Style allows accessor and mutators to be named in like variables: https://google.github.io/styleguide/cppguide.html#Function_Names